src/Makefile

```diff
@@ -80,7 +79,7 @@ CPPFLAGS_TEST  += -DUNIT_TEST \
                   -I$(GTEST_DIR)/include -I$(GMOCK_DIR)/include \
                   -I${ROOT}/modules/cpp-common/test_utils
 
-LDFLAGS += -L${ROOT}/usr/lib
+LDFLAGS += -L${ROOT}/modules/astaire/usr/lib
```

Comment:
I was confused by this at first - why should the FV test need Astaire's include files. I think all these `/modules/astaire` fragments are needed because you are using Astaire's libraries. It would be better to define variables to hold the include and library paths (so we can easily change this in future if needed) and add a comment explaining what's going on.